kafka_consumer/conf.yaml.example

```diff
@@ -41,6 +41,12 @@ instances:
                             # collections. If kafka consumer offsets disabled
                             # this has no effect.
     # zk_prefix: /0.8
+
+    # SSL Configuration
+    # ssl_cafile: /etc/dd-agent/conf.d/kafka.cer
```

Comment:
This path doesn't really exist, can we use something that looks more like a documentation example, like:
```
# ssl_cafile: /path/to/ca/file
```